netdata-installer.sh

```diff
@@ -1035,15 +1037,13 @@ install_react_dashboard() {
   DASHBOARD_PACKAGE_BASENAME="dashboard.tar.gz"
 
   if fetch_and_verify "dashboard" \
-                      "https://github.com/netdata/dashboard/releases/download/${DASHBOARD_PACKAGE_VERSION}/${DASHBOARD_PACKAGE_BASENAME}" \
-                      "${DASHBOARD_PACKAGE_BASENAME}" \
-                      "${tmp}" \
-                      "${NETDATA_LOCAL_TARBALL_OVERRIDE_DASHBOARD}"
-  then
-    if run tar -xf "${tmp}/${DASHBOARD_PACKAGE_BASENAME}" -C "${tmp}" && \
-       copy_react_dashboard "${tmp}/build" && \
-       rm -rf "${tmp}"
-    then
+    "https://github.com/netdata/dashboard/releases/download/${DASHBOARD_PACKAGE_VERSION}/${DASHBOARD_PACKAGE_BASENAME}" \
+    "${DASHBOARD_PACKAGE_BASENAME}" \
+    "${tmp}" \
+    "${NETDATA_LOCAL_TARBALL_OVERRIDE_DASHBOARD}"; then
+    if run tar -xf "${tmp}/${DASHBOARD_PACKAGE_BASENAME}" -C "${tmp}" &&
+      copy_react_dashboard "${tmp}/build" &&
+      rm -rf "${tmp}"; then
```

Comment:
Note that the formatting here was intentional, because it allows someone quickly scanning the code to clearly see the control flow structures.

At minimum please update the continuation lines to be a different indent from the start of the nested `if` statement.

Reply:
I _think_ Github's UI is confusin gyou here. I've checked this version against master and the indnetion is the same.

Reply:
No, your editor is correcting it to look like this on master.

The actual indentation is the same as what your patch is changing it from. See https://github.com/netdata/netdata/blob/master/netdata-installer.sh#L1037

Reply:
Put differently, if this was the same as master, then this hunk shouldn't be in the patch, because there should be _no changes_.

Reply:
I used `less` 🤷‍♂️This isn't really that importantt, I'm parking this for now. The indentation looks the same for me; just I do have all the auto formatters installed in Vim we talked about some time ago in Greece set upp; lately I've noticed that it keeps reformattings things I don't expect as I _thought_ we all agreed to use the same tools?

Reply:
I generally try to use `shfmt`. The issue is that it sometimes leads to readability issues like this. You have to really be paying attention to realize that the nested conditional here is indeed nested and not part of the original condition with things formatted this way.

Reply:
I don't "see" the problem to be honest. I can differentiate the indication just fine.

Perhaps I'm just blind; can you show me where the indentation is wrong?

See attached screenshot with indentation guides turned on.

<img width="750" alt="Screen Shot 2020-05-01 at 01 56 37" src="https://user-images.githubusercontent.com/1290234/80732032-163d1300-8b4f-11ea-89b1-fc59ff10fb9c.png">